Week1/Day2/ExercisesXP/code.py

```diff
@@ -0,0 +1,119 @@
+#Exercise1
+my_fav_numbers = {7,88,24,23,10}
+my_fav_numbers.add(33)
+my_fav_numbers.add(16)
+print(my_fav_numbers)
+my_fav_numbers.remove(10) #we can delete last number only if we know what is last number
```

Comment:
we meant the last number you've added, you're right that set doesn't have order

Reply:
my_fav_numbers.remove(16)